README.md

```diff
@@ -1,68 +1,19 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Getting started
+There are two elements required to run this project, the `server` and `client`.
 
-## Available Scripts
+Open a terminal and run `npm i` to install all dependencies.
 
-In the project directory, you can run:
+### Server
+The server is a Node.js server, which uses the Express framework in order to use Socket.io
 
-### `npm start`
+To start the server, type `nodemon server/server` from the main project directory.
```

Comment:
How come you don't add an entry to your `package.json` scripts, like...
```
"scripts": {
  "start-server": "nodemon ./server/server",
}
```
Then you can run `npm run start-server`, or `yarn start-server`